alembic/versions/46da5adcd1c0_initial_migration.py

```diff
@@ -0,0 +1,49 @@
+"""Initial migration
+
+Revision ID: 46da5adcd1c0
+Revises: 
+Create Date: 2024-11-03 19:59:21.652253
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '46da5adcd1c0'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=127), nullable=False),
+    sa.Column('additional_info', sa.String(length=511), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_city_id'), 'city', ['id'], unique=False)
+    op.create_table('temperature',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('date_time', sa.DateTime(), nullable=False),
+    sa.Column('temperature', sa.Float(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['city_id'], ['city.id'], ondelete='CASCADE'),
```

Comment:
The foreign key constraint on 'city_id' in the 'temperature' table is set to cascade on delete. This means that deleting a city will also delete all related temperature records. Confirm that this behavior is intended for your application.